connect/client.go

```diff
@@ -112,6 +172,24 @@ func removeOrRefreshService(service Service) error {
 	return removeService(service.Name)
 }
 
+// AnnounceSystem announce system via SCC/Registration Proxy
+func AnnounceSystem(distroTgt string, instanceDataFile string) (string, string, error) {
+	fmt.Printf(bold("\nAnnouncing system to " + CFG.BaseURL + " ...\n"))
+	var instanceData []byte
+	if instanceDataFile != "" {
+		var err error
+		instanceData, err = os.ReadFile(instanceDataFile)
```

Comment:
This file is relative to config root.

Reply:
well spotted. UpdateSystem() has the same bug.

Reply:
done